.travis.yml

```diff
@@ -0,0 +1,28 @@
+# :file: .travis.yml
```

Comment:
Accidental re-add?

Reply:
Correct.  Was in my local directory as uncommitted content.  Removing.